### PR TITLE
OHRI-2119: Populate the Recent TPT tiles on the TPT patient summary board

### PR DESCRIPTION
**Pull request by lucyjemutai**
## Requirements

Populate the Recent TPT tiles on the TPT patient summary board

https://github.com/UCSF-IGHS/openmrs-esm-ohri/assets/130601439/5d2e8875-8c82-49df-9d56-955541a3b603

## Screenshots
image

### CODE REVIEW

**Comment by lucyjemutai** [comment]
Hi @pirupius, kindly review this pr. Thanks

File: packages/esm-tb-app/src/config-schema.ts [hunk 1]
```diff
@@ -16,6 +16,8 @@ export const configSchema = {
       tbPatientTracing: '98c938e9-fb3e-4982-ae22-0305cbd12f8c',
       tbProgramEnrollment: '9a199b59-b185-485b-b9b3-a9754e65ae57',
       tbTreatmentAndFollowUp: '1881304a-4854-4927-b0b1-a6231d61e43c',
+      tptCaseEnrollment: '	dc6ce80c-83f8-4ace-a638-21df78542551',
```
Comment (pirupius) [inline packages/esm-tb-app/src/config-schema.ts:19]: Remove the extra space before the uuid

```suggestion
      tptCaseEnrollment: 'dc6ce80c-83f8-4ace-a638-21df78542551',
```

File: packages/esm-tb-app/src/views/tpt/patient-summary/patient-summary.component.tsx [hunk 2]
```diff
@@ -1,19 +1,87 @@
...
-  patientUuid: string;
-}
-
-const TptPatientSummary: React.FC<TptPatientSummaryProps> = ({ patientUuid }) => {
+const TptPatientSummary: React.FC<PatientChartProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
+  const config = useConfig();
+
+  const headerRecentTPT = t('recentTpt', 'Recent TPT Cases');
+  const headerPreviousCases = t('previousCases', 'Previous TPT Cases');
```
Comment (pirupius) [inline packages/esm-tb-app/src/views/tpt/patient-summary/patient-summary.component.tsx:20]: change the key names to include the 3 words `recentTptCases` and `previousTptCases`

File: packages/esm-tb-app/src/views/tpt/patient-summary/patient-summary.component.tsx [hunk 3]
```diff
@@ -1,19 +1,87 @@
...
+import { moduleName } from '../../..';
 
-interface TptPatientSummaryProps {
-  patientUuid: string;
-}
-
-const TptPatientSummary: React.FC<TptPatientSummaryProps> = ({ patientUuid }) => {
+const TptPatientSummary: React.FC<PatientChartProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
+  const config = useConfig();
```
Comment (pirupius) [inline packages/esm-tb-app/src/views/tpt/patient-summary/patient-summary.component.tsx:17]: A nice trick is you can do is destructure this to shorten the names like so 

```
const { obsConcepts, encounterTypes } = useConfig();
```

Then down you can just 
```
encounterTypes.tptCaseEnrollment

OR

obsConcepts.tptEnrollmentDate
```

**Comment by pirupius** [comment]
@lucyjemutai your screenshot seems to have a funny padding being more on the upper card and the lower list being slightly longer.

File: packages/esm-tb-app/src/views/tpt/patient-summary/patient-summary.component.tsx (hunk 3 above)
Comment (lucyjemutai) [reply packages/esm-tb-app/src/views/tpt/patient-summary/patient-summary.component.tsx]: Great! Thanks

**Comment by lucyjemutai** [comment]
@pirupius , could be because i had zoomed the tab. I have attached a short video of how it appears locally.

**Review by pirupius — APPROVED** [review]
Thanks @lucyjemutai

**Pull request merged**